test/e2e/fixture-builder.js

```diff
@@ -1722,14 +1722,6 @@ class FixtureBuilder {
           ticker: 'BNB',
           id: 'bnb-mainnet',
         },
-        'base-mainnet': {
```

Comment:
This fixture merges the default network controller state with this fixture. Similar to Ethereum and Linea, as base is a new default network, we do not need this in the fixture.